spec/services/catalog/order_item_sanitized_parameters_spec.rb

```diff
@@ -70,6 +70,7 @@
           end
 
           it "handles the exception and reraises a StandardError" do
+            expect(Rails.logger).to receive(:error).thrice
```

Comment:
Haha I didn't know `.thrice` existed, thought you would have to do `.exactly(3).times` or something like that. Nice 👍